packages/contracts/deployments/README.md

```diff
@@ -31,9 +34,9 @@
 
 Network : __undefined (chain id: 69)__
 
-| Contract           | Address                                                                                                                         |
-| ------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| OVM_GasPriceOracle | [0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76](https://undefined.etherscan.io/address/0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76) |
+|Contract|Address|
```

Comment:
I'm not sure if typically some sort of linter is run to get the formatting nice in the README.md. Let me know if this should be cleaned up, but currently the `generate-markdown.js` script outputs the README.md like this.

Reply:
Nah I think someone did this manually. This is fine, doesn't make a difference to the final output.

Reply:
Its fine for the script to generate the file like this since it will render without a problem